src/docker/remote.rs

```diff
@@ -34,33 +34,50 @@ fn subcommand_or_exit(engine: &Engine, cmd: &str) -> Result<Command> {
     Ok(engine.subcommand(cmd))
 }
 
+fn posix_parent(path: &str) -> Option<&str> {
```

Comment:
This might need a minor FIXME: `"/".rsplit_once('/')` is `Some(("", ""))`, while `Path::new("/").parent()` is `None`. It doesn't matter here, but should be done.

Reply:
It might be safer to just change this function to:

 ```rust
pub fn posix_parent_path(path: &str) -> Option<&str> {
    std::path::Path::new(path).parent()?.to_str()
}
```

Since that way any nuances (I don't expect any) are properly handled and we're using Rust's tested tooling.